site/content/migrating/01-migrating.md

```diff
@@ -8,7 +8,14 @@ Until we reach version 1.0, there may be occasional changes to the project struc
 
 ### 0.27 to 0.28
 
-`%sapper.scripts%` can be moved to the `<head>` section for slightly better performance because it now uses deferred loading
+* Sapper dropped support for Rollup 0.x.x  ([#1326](https://github.com/sveltejs/sapper/pull/1326)). <br>Any version greater than 1.x is supported but the latest (currently 2.x) is strongly recommended.
+* `<script>` tags will now be loaded with the `defer` attribute ([#1123](https://github.com/sveltejs/sapper/pull/1123)), which means:
+	* IE9 support was dropped since IE9 may interleave deferred script execution. 
```

Comment:
```suggestion
	* IE9 support was dropped since IE9 may interleave deferred script execution.
```

(Sorry, I had a stray space in my first suggestion.)